.neutrinorc.js

```diff
@@ -1,103 +1,37 @@
 module.exports = {
   use: [
-    ['./packages/airbnb-base', {
+    ['./packages/airbnb', {
+      // Excludes are managed vis `.eslintignore` since `exclude` doesn't support globs.
       include: [
         '.*.js',
-        'packages/*/*.js',
-        'packages/*/{src,bin,commands}/*.js',
-        'packages/*/commands/init/*.js',
-        'packages/neutrino/bin/*'
+        'packages/'
       ],
       eslint: {
         baseConfig: {
           extends: [
-            'plugin:react/recommended',
             'prettier',
             'prettier/react'
           ]
         },
         envs: ['browser', 'commonjs', 'node'],
         plugins: [
-          'eslint-plugin-prettier',
-          'eslint-plugin-react'
+          'prettier'
         ],
         rules: {
           // Algebraic and functional types should allow capital constructors without new
           'babel/new-cap': 'off',
-          // Disable necessitating return after a callback
-          'callback-return': 'off',
-          // Allow using class methods with static/non-instance functionality
-          // React lifecycle methods commonly do not use an instance context for anything
-          'class-methods-use-this': 'off',
           // Disallow trailing commas on arrays, objects, functions, et al
           'comma-dangle': ['error', 'never'],
```

Comment:
I'm fine with removing this if you like.

Reply:
Ah so when I looked into `comma-dangle` it was a bit confusing. Airbnb enables it as 'always', the eslint-config-prettier preset then turns it off entirely, and ordinarily the 'prettier/prettier' rule would then turn it back on as 'never' - but we're not using 'prettier/prettier' at the moment.

So removing this line would make it `'comma-dangle': 'off'`. (Which may or may not be what you meant?) :-)

Reply:
Grr, I'm dumb. I guess I was trying to align with:

https://github.com/mozilla-frontend-infra/neutrino-preset-mozilla-frontend-infra/blob/master/lint.js#L43

Thoughts?

Reply:
Zooming out a bit, my thoughts are:
* I like the idea of Neutrino dogfooding it's own linting configs
* To get the most value of dogfooding we should limit the amount we override them (and if things really are an issue, fix the preset directly)
* If we want to use prettier, perhaps medium-long term we should create/use a prettier neutrino preset
* Long term I'd like to see most of the style rules in neutrino-preset-mozilla-frontend-infra go away (I don't really think they belong there - ideally projects should use stock presets and not deviate too much. And I don't think that preset should turn off things like `react/prefer-stateless-function` - the individual projects should make those choices IMO)

Reply:
> And I don't think that preset should turn off things like react/prefer-stateless-function - the individual projects should make those choices IMO

To clarify on this part - I agree that for legacy project some rules (eg `react/prop-types`) are too burdensome - however by hiding the setting of it to `'off'` in neutrino-preset-mozilla-frontend-infra, it means brand new projects (which have no reason to not do things right from the start) don't know to turn it on.

Reply:
Good points all around. Most of the rules in our preset are stylistic choices to improve readability/maintainability, prevent mutation bugs, or get in the way of doing FP.

Let's raise these as issue in that preset.

Reply:
Fwiw the AirBnB style guide are sometimes receptive to suggestions for rule changes. By filing issues against them and linking to those issues when making adjustments, it would (a) show that upstreaming has been attempted, (b) give breadcrumbs to follow for understanding why the deviation :-)

Reply:
Yeah, I should also try and document why we currently have those deviations in the interim.